docs/DeveloperGuide.md

```diff
@@ -320,13 +326,16 @@ also be able to trace functionality bugs more easily if each command class deals
  which extends the main <code>Command</code> class with an index representing that of the item to be marked or
  unmarked as bought in the list.
  
- The process of object creation:
+ The process is as follows:
  1. The <code>Duke</code> class first receives user input from the <code>Ui</code>
- 2. The <code>Duke</code> class then creates a <code>Parser</code> object and calls its <code>parseCommand</code> method
- to instantiate a <code>MarkCommand</code> or <code>UnmarkCommand</code> object based on the user input
- 3. The <code>Duke</code> class then calls the <code>execute</code> method of the <code>MarkCommand</code> or 
- <code>UnmarkCommand</code> command object. This calls the <code>markAsBought</code> or <code>unmarkAsBought</code>
- method of the <code>shoppingList</code> object with the specified index.
+ 2. The <code>Duke</code> class then creates a <code>Parser</code> object and calls its
+ <code>Parser#parseCommand()</code> method to instantiate a <code>MarkCommand</code> / <code>UnmarkCommand</code>
+ object based on the user input
+ 3. The <code>Duke</code> class then calls the <code>MarkCommand#execute()</code> /
+ <code>UnmarkCommand#execute()</code> method. 
+ 4. The <code>MarkCommand#execute()</code> / <code>UnmarkCommand#execute()</code> calls the
+ <code>ShoppingList#markAsBought()</code> / <code>ShoppingList#unmarkAsBought()</code> method with the
+ specified index.
```

Comment:
Maybe can just use "Duke" instead of "The Duke class", since most of the other sections just use Duke. Also maybe can omit "the" for "the MarkCommand#execute" since for your other sections you never use "the". Can follow your reset budget section

Reply:
Ok, followed the format! Thanks!